### DISCUSSION TITLE
I found an icon css bug

### DISCUSSION THREAD

**Discussion by renfei** [community/community#3732] [Other Feature Feedback, Questions, & Ideas]
I think I found a CSS bug:
![image](https://user-images.githubusercontent.com/6532952/114039863-afbf8980-98b5-11eb-83a0-45ea18204244.png)
![image](https://user-images.githubusercontent.com/6532952/114039879-b2ba7a00-98b5-11eb-8a3c-f39bbdf9d53a.png)
![image](https://user-images.githubusercontent.com/6532952/114039899-b5b56a80-98b5-11eb-9c7d-5dcbec21d198.png)

My browser:
![image](https://user-images.githubusercontent.com/6532952/114039909-b817c480-98b5-11eb-90f6-a78e9d3028ce.png)

**Comment by bastiengrignon** [comment]
Same for me !
It's just a `z-index` issue

![B1lRsoAcOL](https://user-images.githubusercontent.com/41306995/114098078-29a94000-98c1-11eb-9820-33bc2dca0cc8.gif)
